config/local_development.php

```diff
@@ -38,4 +38,14 @@
     'super_admin' => [
         'email' => 'sampleadmin@advising.app',
     ],
+    'internal_users' => [
+        'emails' => env('DEMO_INTERNAL_USER_EMAILS') ? explode(',', env('DEMO_INTERNAL_USER_EMAILS')) : null,
+    ],
+    'twilio' => [
+        'account_sid' => env('TWILIO_ACCOUNT_SID'),
+        'auth_token' => env('TWILIO_AUTH_TOKEN'),
+        'from_number' => env('TWILIO_TEST_FROM_NUMBER', env('TWILIO_FROM_NUMBER', env('TWILIO_PHONE_NUMBER'))),
+        'to_number' => env('TWILIO_TEST_TO_NUMBER', env('TWILIO_TO_NUMBER')),
```

Comment:
What is the purpose of having these in a separate local development config? Do we have config for the twilio integration itself?

I expect that it make sense to not need to duplicate things like this. Local development settings really only need to be like `enable_test_sender` shouldn't it?

Reply:
The original three test settings were created awhile ago to help with development and were being used prior to this. 

I added `account_sid` and `auth_token` to speed up local development after I copied the keys 30+ times. I relocated the other configs to colocate them. I thought this was prudent given the rest of the configs in `services` are production configs and these only serve a purpose in testing, but they were left in the there after we moved the production configs into the UI. None of the

The `local_development` config was only used before to set the default super admin email, but I thought it would be a good place to centralize our other local development configs.

Reply:
Got it, so these will act as an override the each tenant uses so you don't have to go put them in manually into the integration setting?

Reply:
Yeah, if you look at the `--local` command option you see I only call the seeder after everything is created like we've done with the other options: 

```php
if ($this->option('yes') || $this->option('local') || $this->confirm('Would you like to seed local development data?')) {
    Artisan::call(
        command: "tenants:artisan \"db:seed --database={$database} --class=LocalDevelopmentSeeder\" --tenant={$tenant->id}",
        outputBuffer: $this->output,
    );
}
```
My hope is that we can just keep adding things to the seeder and config and we don't have to setup the integrations every time for each tenant. And in general just makes it a faster setup.